solidity/contracts/transformers/wstETHTransformer.sol

```diff
@@ -23,10 +23,16 @@ contract wstETHTransformer is BaseTransformer {
   }
 
   /// @inheritdoc ITransformer
-  function calculateTransformToUnderlying(address, uint256 _amountDependent) external view returns (UnderlyingAmount[] memory) {}
+  function calculateTransformToUnderlying(address, uint256 _amountDependent) external view returns (UnderlyingAmount[] memory) {
+    uint256 _amountUnderlying = stETH.getPooledEthByShares(_amountDependent);
+    return _toSingletonArray(stETH, _amountUnderlying);
+  }
```

Comment:
If this is a `wstETH <=> stETH` Transformer, then `wstETH` will be the dependent and `stETH` will be the underlying, right? If that is the case, then why are we using `getPooledEthByShares` that returns `ETH` as underlying instead of something denominated on`stETH`?
Clearly, there is something I am missing here.

Reply:
These are very valid questions, but I'm doing exactly what wstETH is doing [here](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.6.12/WstETH.sol#L90). Same for the question below

Instead of using wstETH, I'm using stETH directly, because that would allow me to save some gas